splunk_add_on_ucc_framework/schema/schema.json

```diff
@@ -79,6 +79,10 @@
         "description": {
           "type": "string"
         },
+        "iconFileName": {
```

Comment:
Above you mentioned that `iconFileName` has a default - should we reflect it is schema as well?